CoMD/src-threaded/ljForce.cpp

```diff
@@ -166,14 +170,21 @@ int ljForce(SimFlat* s)
    const real_t eShift = POT_SHIFT * rCut6 * (rCut6 - 1.0);
 
    {
-     RAJA::kernel<ljForcePolicy>(
+#ifdef DO_CUDA
+     RAJA::kernel<forcePolicyGPU>(
+#else
+     RAJA::kernel<forcePolicy>(
+#endif
        RAJA::make_tuple(
          *s->isLocalSegment,                // local boxes
          RAJA::RangeSegment(0,27),          // 27 neighbor boxes
          RAJA::RangeSegment(0, MAXATOMS),   // atoms i in local box
          RAJA::RangeSegment(0, MAXATOMS) ), // atoms j in neighbor box
-
+#ifdef DO_CUDA
+       [=] RAJA_DEVICE (int iBoxID, int nghb, int iOff, int jOff) {
```

Comment:
I wonder if we can replace RAJA_DEVICE here with a macro that is null if we are not using cuda so we wouldn't have the ifdef here but rather somewhere outside the kernel.